polyfills/Map/polyfill.js

```diff
@@ -0,0 +1,124 @@
+(function(global) {
+
+
+	// Deleted map items mess with iterator pointers, so rather than removing them mark them as deleted. Can't use undefined or null since those both valid keys so use a private symbol.
+	var undefMarker = {};
```

Comment:
We could create a symbol using `Symbol()` because we have Symbol declared as a dependency for this polyfill.